### PR TITLE
Add page insets to CarouselGridLayout

### PR DESCRIPTION
**Pull request by Clstroud**
* Enables carousel collection views to be edge-to-edge in a future revision
This is currently a breaking change that will be rectified as a byproduct of the forthcoming refactor work

### CODE REVIEW

File: Vocable/Common/CarouselGridLayout.swift [hunk 1]
```diff
@@ -110,14 +114,14 @@ class CarouselGridLayout: UICollectionViewLayout {
...
+            pageContentSize.width > 0 else { return 0 }
+        let index = Int(collectionView.bounds.midX / pageContentSize.width)
         return index
     }
 
     override var collectionViewContentSize: CGSize {
-        var size = sectionContentSize.applying(.init(scaleX: CGFloat(numberOfSections), y: 1))
-        size.width -= interItemSpacing // The last page does not have trailing padding
+        let size = sectionContentSize.applying(.init(scaleX: CGFloat(numberOfSections), y: 1))
+//        size.width -= _interPageSpacing // The last page does not have trailing padding
```
Comment (stechkev) [inline Vocable/Common/CarouselGridLayout.swift:124]: was this left in on purpose?

Comment (stechkev) [inline Vocable/Common/CarouselGridLayout.swift]:
```suggestion
```

**Review by stevefosterwta — APPROVED** [review]
👍

**Comment by stechkev** [comment]
Test approved. breaking the app only on carousels in the expected way

**Pull request merged**